src/Prefixer.js

```diff
@@ -82,12 +89,15 @@ export function prefix (value, length) {
 						return ~indexof(value, 'stretch') ? prefix(replace(value, 'stretch', 'fill-available'), length) + value : value
 				}
 			break
+		// grid-(column|row)
+		case 5152: case 5920:
+			return replace(value, /(.+?):(\d+)(\s*\/\s*(span)?\s*(\d+))?(.*)/, function (_, a, b, c, d, e, f) { return (MS + a + ':' + b + f) + (c ? (MS + a + '-span:' + (d ? e : +e - +b)) + f : '') + value })
```

Comment:
@SukkaW I updated this, can you review? does it still handle the original cases you had in mind?

Reply:
Since the test cases passed, so I assume it does.